site/content/en/docs/tasks/run_python_jobs.md

```diff
@@ -277,3 +277,95 @@ hello world
 
 You can further customize the job, and can ask questions on the [Flux Operator issues board](https://github.com/flux-framework/flux-operator/issues).
 Finally, for instructions for how to do this with YAML outside of Python, see [Run A Flux MiniCluster](/docs/tasks/run_flux_minicluster/).
+
+### MPI Operator Job
+
+For this example, we will be using the [MPI Operator](https://www.kubeflow.org/docs/components/training/mpi/)
+to submit a job, and specifically using the [Python SDK](https://github.com/kubeflow/mpi-operator/tree/master/sdk/python/v2beta1) to do this easily. Given our Python environment created in the [setup](#before-you-begin), we can install this Python SDK directly to it as follows:
+
+```bash
+git clone --depth 1 https://github.com/kubeflow/mpi-operator /tmp/mpijob
+cd /tmp/mpijob/sdk/python/v2beta1
+python setup.py install
+cd -
+```
+
+Importantly, the MPI Operator *must be installed before Kueue* for this to work! Let's start from scratch with a new Kind cluster.
+We will also need to [install the MPI operator](https://github.com/kubeflow/mpi-operator/tree/master#installation) and Kueue. Here we install
+the exact versions tested with this example:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubeflow/mpi-operator/master/deploy/v2beta1/mpi-operator.yaml
```

Comment:
why not use a released version too? https://github.com/kubeflow/mpi-operator/releases/tag/v0.4.0

Reply:
Only because someone then has to explicitly update it. Would you like this changed to a released version?

Reply:
unreleased versions have higher chances of being broken, so it's not a good practice in general. You can always put a note along the lines:

Check the [mpi-operator release page](https://github.com/kubeflow/mpi-operator/releases) for alternate versions.

Reply:
I definitely agree with you there! Alrighty, pushed.